CHANGELOG.md

```diff
@@ -22,10 +22,10 @@ The below sections record the changes for each API version in each (pre-)release
 
 NOTE: SimpleEdgeDiscovery is part of the EdgeCloud API family, and was previously developed within the [EdgeCloud](https://github.com/camaraproject/EdgeCloud) repository. Two releases were made within EdgeCloud, v0.81 and r0.9.3-rc. These releases also include the other EdgeCloud APIs and materials, so the changelog below highlights only the changes related to SimpleEdgeDiscovery. Further information and links to the releases in EdgeCloud can be found below.
 
-# v1.1.0-rc.1
+# v1.1.0-r2.1
 
 This release candidate contains the definition and documentation of
-* simple-edge-discovery v1.1.0-rc.1
+* simple-edge-discovery v1.1.0-r2.1
```

Comment:
The API version is v1.1.0-rc.1 ... it is independent from the release numbering!

```suggestion
* simple-edge-discovery v1.1.0-rc.1
```